google_analytics/ga_homepage_report.py

```diff
@@ -37,6 +37,7 @@
 
 import argparse
 
+# pip install --upgrade google-api-python-client
```

Comment:
@clintonb merged this before github showed me your comments.  I'll add this to `requirements.txt`